src/index.js

```diff
@@ -79,19 +79,17 @@ render(
   <Provider store={store}>
     <ReactReduxFirebaseProvider {...rrfProps}>
       <Router basename="/kansassmittys">
-        <Switch>
-          <Route path="/account/register" component={Register} />
-          <Route path="/account/profile" component={Profile} />
-          <Route path="/account/questions" component={Questions} />
-          <Route path="/account/code-of-conduct" component={CodeOfConduct} />
-          <Route path="/login" component={Login} />
-          <WithLoadedUsers>
-            <Switch>
-              <Route path="/account/edit" component={EditProfilePage} />
-              <Route path="/" component={App} />
-            </Switch>
-          </WithLoadedUsers>
-        </Switch>
+        <WithLoadedUsers>
```

Comment:
Would it take long to extract the password to a separate page and have it out of this wrapper?